{{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -5,7 +5,7 @@ title: "[BUG]: "
 labels:
   - bug
 assignees:
-  - "{{cookiecutter.github_username}}"
+  - "{{cookiecutter.github_user_or_org}}"
```

Comment:
Can an organisation be assigned as bug ticket?

Reply:
Do you mean can an organisation the assignee on a GitHub issue? I don't think so.

Reply:
Yes, that's what I meant. 8 am pre-coffee grammar. Sorry.